ibmcloud/README.md

```diff
@@ -160,6 +160,60 @@ You need to build a pod VM image for peer pod VMs. A pod VM image contains the f
 
 The build scripts are located in [ibmcloud/image](./image). The prerequisite software to build a pod VM image is already installed in the worker node by [the Ansible playbook](terraform/cluster/ansible/playbook.yml) for convenience.
 
+You need to configure Cloud Object Storage (COS) to upload your custom VM image.
+
+https://cloud.ibm.com/objectstorage/
+
+First, create a COS service instance if you have not create one. Then, create a COS bucket with the COS instance. The COS service instance and bucket names are necessary to upload a custom VM image.
+
+Next, you need to grant access to COS to import images as described at [https://cloud.ibm.com/docs/vpc?topic=vpc-object-storage-prereq&interface=cli](https://cloud.ibm.com/docs/vpc?topic=vpc-object-storage-prereq&interface=cli).
+
+ibmcloud login -r jp-tok -apikey $api_key
```

Comment:
missed ``` for L171-174

Reply:
Thanks for spotting that. I added the missing code block.